ifrs17/Import/Importers.ipynb

```diff
@@ -503,18 +503,33 @@
         {
             "cell_type": "code",
             "source": [
-                "public async Task<ImportArgs[]> GetAllArgsAsync(ImportArgs args, IDataSource dataSource, string format, IDataSource targetDataSource)",
+                "public async Task<ImportArgs[]> GetAllArgsAsync(ImportArgs args, IDataSource dataSource, string format)",
                 "\n{",
                 "\n    ImportArgs[] allArgs;",
                 "\n    switch(format)",
                 "\n    {",
-                "\n        case ImportFormats.YieldCurve : ",
+                "\n        case ImportFormats.Cashflow:",
+                "\n        case ImportFormats.Actual: ",
+                "\n        case ImportFormats.Opening:",
+                "\n        case ImportFormats.SimpleValue:",
+                "\n        {",
+                "\n            if(args.Scenario != null) return args.RepeatOnce().ToArray();",
+                "\n            var secondaryArgs = await dataSource.Query<PartitionByReportingNodeAndPeriod>() ",
+                "\n                .Where(x => x.ReportingNode == args.ReportingNode && x.Year == args.Year && x.Month == args.Month && x.Scenario != null)",
+                "\n                .Select(x => new ImportArgs(x.ReportingNode, x.Year, x.Month, default(Periodicity), x.Scenario, ImportFormats.Cashflow)).ToArrayAsync();",
```

Comment:
ImportFormat is set to ImportFormats.Cashflow in every case? also when ImportFormats is Actual and Opening?

Reply:
I believe this is correct because the secondary args are all those scenario args that come into play when importing something Best Estimate. In this implementation, scenario calculations have import format cashflow.

Reply:
This line has not been changed by this PR, that is, same behavior was there prior to this PR.

Reply:
Having FormatCashflow computes from RawVar and it does not compute Actuals such as Advance and Overdue. 
If you re-import an Actual BE and you have already imported an Actual scenario you need to recompute the scenario performing calculation of Advance and Overdue. Now I think you would re-compute them.

Reply:
Do you suggest to compute from Actual in case of a (re-)import of an Actual BE?

Reply:
ok, I am convinced and I agree with you: the import format in the case of BE actuals need to be ImportFormats.Actual. I have applied the change to achieve this behavior in the lines you picked up.